Desktop/property-pro-app/src/components/Midsection/Midsection.js

```diff
@@ -0,0 +1,82 @@
+import React from "react";
+import "./midsection.css";
+
+export default function Midsection() {
+  return (
+    <div className="mid">
+      <div className="boxcontainer">
+        <div className="buy">
+          <div className="icon">
+            <img
+              src="https://img.icons8.com/carbon-copy/100/000000/home.png"
+              alt="house.img"
+            />
+          </div>
+          <div className="middleheader">
+            <p className="caption">
+              {" "}
+              <b> BUY A HOME </b>{" "}
+            </p>
+            <p>
+              {" "}
+              Find your place with an immersive photo and top listing, including
+              things you won't find anywhere else.
+            </p>
+          </div>
+          <div className=" search">
+            <button className="searchbutton">Search Home </button>
+          </div>
+        </div>
+      </div>
+
+      <div className="boxcontainer">
+        <div className="buy">
+          <div className="icon">
+            {" "}
+            <img
+              src="https://img.icons8.com/carbon-copy/100/000000/home.png"
```

Comment:
You may want to download this image to your local system rather than tapping it from a url because the image from the url may not always be available and you dont want that for your app.